packages/0x.js/package.json

```diff
@@ -60,15 +60,14 @@
         "node": ">=6.0.0"
     },
     "devDependencies": {
-        "@0xproject/abi-gen": "^0.2.9",
```

Comment:
This is used

Reply:
It actually isn't... We go up the directory path and directly call the `abi-gen` package... Think we shouldn't leave this here unless it's **actually** used.